http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -49,13 +51,15 @@ private FeignClientFactory(
             Decoder decoder,
             ErrorDecoder errorDecoder,
             Function<Optional<SSLSocketFactory>, Client> clientSupplier,
-            BackoffStrategy backoffStrategy) {
+            BackoffStrategy backoffStrategy,
+            Request.Options options) {
```

Comment:
Yeah, I cut the release with this as is, we can add the other method later, as necessary.